src/cute_app.cpp

```diff
@@ -233,11 +233,11 @@ CF_Result cf_make_app(const char* window_title, CF_DisplayID display_id, int x,
 	// Turn on high DPI support for all platforms.
 	options |= SDL_WINDOW_HIGH_PIXEL_DENSITY;
 	
-	if (SDL_Init(sdl_options)) {
+	if (!SDL_Init(sdl_options)) {
```

Comment:
SDL_Init and SDL_InitSubsystem (Along with some of the other wrappers around InitSubsystem) seem to return true on success now.